### PR TITLE
Move security and privacy requirements into DID methods

### LINKED ISSUE #107: Normative language in Security Considerations
There's a bunch of normative language about the security considerations of DID methods, in the Security Considerations section (which is traditionally, and labelled as, a non-normative section). Should this be moved to the DID Methods section?

### PR DESCRIPTION
**Pull request by rhiaro**
Normative language about security and privacy requirements for DID methods is now in Security Requirements and Privacy Requirements subsections of the DID methods section, rather than in the (non-normative) Security Considerations section. 

Fixes #107

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/112.html" title="Last updated on Nov 27, 2019, 9:12 AM UTC (9917b12)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/112/a8d0c28...9917b12.html" title="Last updated on Nov 27, 2019, 9:12 AM UTC (9917b12)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2318,68 +2256,147 @@ <h2>
...
+identified as well. 
+      </p>
+
+      <p>
+If the protocol incorporates cryptographic protection mechanisms,
+it should be clearly indicated which portions
 of the data are protected and what the protections are (i.e.,
 integrity only, confidentiality, and/or endpoint authentication,
-etc.). Some indication should also be given to what sorts of attacks
+etc.). Some indication should also be given of to what sorts of attacks
```
Comment (TallTed) [inline index.html]:
```suggestion
etc.). Some indication should also be given of the sorts of attacks to which
```

File: index.html [hunk 2]
```diff
@@ -2318,68 +2256,147 @@ <h2>
...
+    <h1>
+Security Considerations
+    </h1>
+
+    <p>
+NOTE TO IMPLEMENTERS: During the Implementer’s Draft stage, this
+section focuses on security topics that should be important in early
+implementations. The editors are also seeking feedback on threats and
+threat mitigations that should be reflected in this section or
+elsewhere in the spec. As the root identifier records for <a>decentralized
```
Comment (TallTed) [inline index.html]: This isn't quite a sentence. I can't tell what it's meant to say, so I can't provide a suggested revision --
```
As the root identifier records for <a>decentralized
identifiers</a> (DIDs) and <a>DID documents</a> are a vital component of
<a>decentralized identity management</a>.
```

File: index.html [hunk 3]
```diff
@@ -2318,68 +2256,147 @@ <h2>
...
+identifiers</a> (DIDs) and <a>DID documents</a> are a vital component of
+<a>decentralized identity management</a>. They are also the foundational
+building blocks of <a>decentralized public key infrastructure</a> (DPKI)
+as an augmentation to conventional X.509 certificates. As such, <a>DIDs</a> are
+designed to operate under the general Internet threat model used by
+many IETF standards. We assume uncompromised endpoints, but allow
+messages to be read or corrupted on the network. Protecting against an
+attack when a system is compromised requires external key-signing
+hardware. See also Section <a href="#key-revocation-and-recovery"></a>
```
Comment (TallTed) [inline index.html]:
```suggestion
hardware. See also <a href="#key-revocation-and-recovery"></a>.
```

File: index.html [hunk 4]
```diff
@@ -2318,68 +2256,147 @@ <h2>
...
+regarding key revocation and recovery. For their part, the <a>DLTs</a> hosting
```
Comment (TallTed) [inline index.html]:
```suggestion
For their part, the <a>DLTs</a> hosting
```

File: index.html (hunk 2 above)
Comment (rhiaro) [reply index.html]: I agree. I suspect the whole paragraph is old and might not be relevant any more, but rewriting or removing it probably needs to be a different PR

Comment (TallTed) [reply index.html]: opened #115 to track

**Comment by rhiaro** [comment]
I completely messed up that rebase, I'm going to close this and open a clean PR

**Pull request closed without merging**